CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ This creates the faas-cli for Mac, Windows, Linux x64, Linux ARMHF and Linux ARM
 * Get the vendoring tool called `vndr`
 
 ```
-$ go install github.com/LK4D4/vndr
+$ go get github.com/LK4D4/vndr
```

Comment:
Why did this change? What was wrong with the old version?

Reply:
It didn't work for me with install, but did after `go get`, perhaps it is just a local error for my machine?  I will change it back, I didn't intend to include it.